.travis.yml

```diff
@@ -1,7 +1,11 @@
 sudo: false
 language: python
+cache:
+    directories:
+        - $TRAVIS_BUILD_DIR/eggs
 python:
   - "2.7"
+  - "3.6"
```

Comment:
`tox.ini` also tests on Python 3.4 and 3.5. I think this should be unified. Python 3.4 does not need to be supported as nobody will start using it for migration to Python 3. Python 3.5 could be interesting.